st2api/tests/unit/controllers/v1/test_executions_simple.py

```diff
@@ -281,6 +281,47 @@ def test_post_with_st2_context_in_headers_failed(self):
         self.assertEqual(resp.status_int, 400)
         self.assertIn('Unable to convert st2-context', resp.json['faultstring'])
 
+    def test_re_run_success(self):
+        # Create a new execution
+        post_resp = self._do_post(LIVE_ACTION_1)
+        self.assertEqual(post_resp.status_int, 201)
+        execution_id = self._get_actionexecution_id(post_resp)
+
+        # Re-run created execution (no parameters overrides)
+        data = {}
+        re_run_resp = self.app.post_json('/v1/actionexecutions/%s/re_run' %
```

Comment:
Yeah, I personally never liked "actionsexecutions", but that's a better change.

I'm fine with changing just this one controller and change other in a separate PR.